src/test/groovy/transform/stc/BugsSTCTest.groovy

```diff
@@ -650,4 +650,42 @@ Printer
             List tmp = (List) m.get("some_key_here")     // <---  actually groovy crashes here!!
         '''
     }
+
+    // GROOVY-7416
+    void testMethodsFromInterfacesOfSuperClassesShouldBeVisible() {
+        assertScript '''
+            import groovy.transform.CompileStatic
+
+            interface SomeInterface {
+                void someInterfaceMethod()
+            }
+
+            abstract class AbstractSuperClass implements SomeInterface {}
+
+            @CompileStatic
```

Comment:
Are the CompileStatic annotations here and in the following test necessary? BugsSTCTest extends StaticTypeCheckingTestCase.